### PR TITLE
feat(refactor): Moved sub

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR adds | fixes:

- See above

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Review by jemten — APPROVED** [review]
👍

File: lib/MIP/Analysis.pm [hunk 1]
```diff
@@ -813,6 +814,56 @@ sub set_recipe_bwa_mem {
...
+
+    my $tmpl = {
+        analysis_recipe_href => {
+            default     => {},
+            defined     => 1,
+            required    => 1,
+            store       => \$analysis_recipe_href,
+            strict_type => 1,
+        },
+        sample_info_href => {
```
Comment (ramprasadn) [inline lib/MIP/Analysis.pm:838]: deeptrio_mode => {	
            required    => 1,	
            store       => \$deeptrio_mode,	
            strict_type => 1,	
        },	
This bit is missing from here...

Comment (henrikstranneheim) [reply lib/MIP/Analysis.pm:845]: Fixed

**Review by jemten — APPROVED** [review]
👍

**Pull request merged**